src/coord/src/catalog/builtin.rs

```diff
@@ -926,8 +926,8 @@ lazy_static! {
         id: GlobalId::System(4047),
         index_id: GlobalId::System(4048),
         // Note that the `system_table_enabled` field of PersistConfig (hooked
-        // up to --persistent-system-tables) also has to be true for this to be
-        // persisted.
+        // up to --disable_persistent_system_tables_test) also has to be true
```

Comment:
technically the flag is externally visible as kebab-case: `--disable-persistent-system-tables-test` but these are internal docs and I don't think worth going through all of CI again for.